vllm/outputs.py

```diff
@@ -92,7 +93,7 @@ class RequestOutput:
     def __init__(
         self,
         request_id: str,
-        prompt: Optional[str],
+        prompt: Optional[PromptType],
```

Comment:
Sorry for the after-the-fact comment and for missing in review .. I think this one might not be the right change ... it breaks things in other places and is also part of the API so probably shouldn't change in this way.

I think probably the reason it was changed was due to the recent beam_search logic which still needs a lot of work and I think incorrectly is assigning a var of PromptType to this.

I can address this in another PR that I'm working on right now.

Reply:
@njhill sorry about that. I think it was OK at one point, and then when I rebased on `main` some new errors started happening and I didn't fix them before merge. Let me know if you'd like me to just revert this if you want it fixed quickly.

Reply:
Thanks @russellb no problem I have a PR almost ready